pkg/apis/feature/flag_names.go

```diff
@@ -15,3 +15,7 @@ limitations under the License.
 */
 
 package feature
+
+const (
+	KReferenceGroup = "kreference-group"
```

Comment:
why is this not defined in PKG?

Reply:
given the fact we are adding the change to PKG, I think the feature flag lives at the Knative level inside of pkg, then networking can use this feature flag as well.

Reply:
I think the problem here is that we're still not in the shape to have a single feature flag CM, with flags defined for the project as a whole... Maybe worth to be discussed in this context https://groups.google.com/g/knative-dev/c/M_10Rh_30ho/m/dWD7hpzcAAAJ?

Reply:
In pkg, in order to disable the `KReference.Group` field, there is a specific context flag, so we should be good to go: https://github.com/knative/pkg/pull/2127/files#diff-ee7cca2a46e1af5b2aa2a11e1ae0e064e24bf589126d86dc0eb584e0df724966R114-R126